modules/op-running-a-pipeline.adoc

```diff
@@ -42,7 +42,7 @@ $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=pipelines-vote-ui \
     -p git-url=https://github.com/openshift/pipelines-vote-ui.git \
-    -p IMAGE='image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/pipelines-vote-ui' \
+    -p IMAGE='image-registry.openshift-image-registry.svc:5000/$pipelines-tutorial/pipelines-vote-ui' \
```

Comment:
Don't know if this  is also supposed to be updated. It wasn't called out in GH Issue-5441 but assuming it should apply here, too.